src/lib/components/Actions.svelte

```diff
@@ -32,23 +24,12 @@
 		<p slot="tip-text">Select images folder</p>
 	</Tooltip>
 
-	<Tooltip hidden={!$galleryStore.selectedTag}>
-		<Button
-			slot="tip-trigger"
-			variant={'outline'}
-			on:click={() => dispatch('deleteTag')}
-			class="rounded-lg bg-gray-200 px-3"
-		>
-			<Fa icon={faTrashAlt} />
-		</Button>
-		<p slot="tip-text">Delete tag</p>
-	</Tooltip>
-
 	<Tooltip>
 		<Button
 			slot="tip-trigger"
 			variant={'outline'}
-			disabled={!$galleryStore.selectedImage}
+			disabled={false} 
```

Comment:
This will open the delete modal without the selected image, so which leads to no action from the modal, which is ambiguous and a bad UX.